src/test/kotlin/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTServiceTest.kt

```diff
@@ -141,7 +141,8 @@ class InformatieObjectenRESTServiceTest : BehaviorSpec() {
                             zaak.uuid,
                             documentReferentieId,
                             false,
-                            restEnkelvoudigInformatieobject
+                            restEnkelvoudigInformatieobject,
+                            null
```

Comment:
Of course we would also like to test it with RESTFileUpload. I think it's good to add a test for that here too right?

And you could also consider changing the integration test accordingly but I guess that is a next step?

Reply:
Yep. Wanted to check first if this is the right direction

Reply:
yeah, looks good I think!

Reply:
Added additional case for create and covered the happy-path for update